src/fetch/FetchExample.re

```diff
@@ -0,0 +1,93 @@
+type tDog = string;
```

Comment:
I think the convention is to not distinguish type names in any way, since they live in a different namespace and won't collide with anything. So just `dog` should be fine. Keep it simple.